pkg/apis/core/v1beta1/types_common.go

```diff
@@ -32,6 +32,8 @@ const (
 	// ErrorInfraDependencies indicates that the last error occurred due to dependent objects on the infrastructure level.
 	// It is classified as a non-retryable error code.
 	ErrorInfraDependencies ErrorCode = "ERR_INFRA_DEPENDENCIES"
+	// ErrorRetryableInfraDependencies indicates that the last error occurred due to dependent objects on the infrastructure level, but operation should be retried.
+	ErrorRetryableInfraDependencies ErrorCode = "ERR_RETRYABLE_INFRA_DEPENDENCIES"
```

Comment:
Can you update https://github.com/gardener/gardener/blob/master/docs/usage/shoot_status.md#error-codes?
Can you add the error code also for the internal type and v1alpha1 - https://github.com/gardener/gardener/blob/f86f6e4ce3ff57545551e0331411e654aa7da33f/pkg/apis/core/types_common.go#L22 and https://github.com/gardener/gardener/blob/f86f6e4ce3ff57545551e0331411e654aa7da33f/pkg/apis/core/v1alpha1/types_common.go#L20?